src/main/java/com/stripe/model/Person.java

```diff
@@ -132,16 +144,62 @@ protected String getInstanceUrl() {
     List<String> pastDue;
   }
 
-  // TODO: Move this to a top level class instead of duplicating
-  // with the LegalEntity resource
   @Getter
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class Verification extends StripeObject {
     String details;
     String detailsCode;
-    String document;
-    String documentBack;
+    VerificationDocument document;
+    // TODO: Figure out how to handle the change in shape and deprecate the next 2
+    //String document;
+    //String documentBack;
```

Comment:
@ob-stripe Can you point me to the right solution to handle this? I know we do that in the library today, I'm just not sure where :p

Reply:
We'll need to write yet another custom deserializer :(

This is basically the same case as `evidence` on `Dispute` objects. You'll need to do something like this:
1. Name the new `VerificationDocument` attribute `documentSubObject`.
2. Write a custom deserializer (similar to [`DisputeDataDeserializer`](https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/DisputeDataDeserializer.java)) that looks at the shape of the `document` attribute. If it's a string, then use its value to fill `document`. If if's a hash, then deserialize it as a `VerificationDocument` and fill `documentSubObject`.